CONTRIBUTORS.md

```diff
@@ -32,6 +32,48 @@ This project and everyone participating in it is governed by this [Code of Condu
 - For consensus breaking changes, branch off of the `next` branch.
 - For hotfixes, branch off of `master`.
 
+### GitHub Tags
+We use tags on PRs and issues to categorize them. You should try to select a tag from each category below if possible. 
+
+**Timeline- and consensus- based tags:**
+- Next non-breaking change => `ship now`
+- Next breaking change => `ship next`
+- Future change, in the work pipeline => `ship future`
```

Comment:
Like above, I'd prefer `future stacks` or `future consensus`

Reply:
"future consensus" +1

Reply:
So - ship future was supposed to encapsulate both consensus and non-consensus breaking changes. 

It's why I have `consensus-critical` as a totally separate tag.